package.json

```diff
@@ -80,6 +82,7 @@
   },
   "scripts": {
     "clean": "rm -rf ./dist",
+    "postinstall": "patch-package",
```

Comment:
I think we should run this manually in CI instead of the `postinstall` script. Since the post install would be also executed when users install the CLI.

Reply:
I just wan't to make sure this runs only for the development/tests, but not in prod when users install the wizard.

Reply:
Thank you for all the feedback @krystofwoldrich 🙇 
I was [initially thinking](https://github.com/getsentry/sentry-wizard/pull/889#discussion_r2060085493) of applying in all builds but since we have no reports of this issue it makes sense to keep it only for ci testing. [I reverted the dependencies additions (of both `patch-package` and. `postinstall`)](https://github.com/getsentry/sentry-wizard/pull/889/commits/152dee5f6d407a22e930a923ee209a89f434c101) and [just apply the patch before building on CI](https://github.com/getsentry/sentry-wizard/pull/889/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721).